src/main/java/Internship.java

```diff
@@ -0,0 +1,91 @@
+import java.time.YearMonth;
+import java.time.format.DateTimeFormatter;
+
+public class Internship {
+    private String id;
+    private String role;
+    private String company;
+    private YearMonth startDate;
+    private YearMonth endDate;
+    private String skills;
+    private String status;
+
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/yy");
+
+    public Internship (String role, String company, String start, String end, String skills, String status) {
+        this.role = role;
+        this.company = company;
+        this.startDate = (start == null || start.isEmpty()) ? YearMonth.parse("01/00", formatter) : YearMonth.parse(start, formatter);
+        this.endDate = (end == null || end.isEmpty()) ? YearMonth.parse("01/00", formatter) : YearMonth.parse(end, formatter);
```

Comment:
I think you can break up lines 18 and 19, they are more than 120 characters long